src/hotspot/cpu/arm/arm.ad

```diff
@@ -1886,6 +1874,36 @@ operand immP0() %{
   interface(CONST_INTER);
 %}
 
+// Pointer Immediate
```

Comment:
Why do you introduce immN operands for arm32?

Reply:
This was accidental. I'll remove it.

Reply:
Ok. The ARM32 changes look good to me now!